src/main/java/com/salessparrow/api/config/MemcachedConfig.java

```diff
@@ -0,0 +1,66 @@
+
+package com.salessparrow.api.config;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import com.salessparrow.api.lib.globalConstants.CacheConstants;
+import com.salessparrow.api.utility.Memcached;
+
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.CachingConfigurer;
+import org.springframework.cache.interceptor.CacheErrorHandler;
+import org.springframework.cache.interceptor.CacheResolver;
+import org.springframework.cache.interceptor.KeyGenerator;
+import org.springframework.cache.interceptor.SimpleCacheErrorHandler;
+import org.springframework.cache.interceptor.SimpleKeyGenerator;
+import org.springframework.cache.support.SimpleCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MemcachedConfig implements CachingConfigurer {
+
+  @Override
+  @Bean
+  public CacheManager cacheManager() {
+    SimpleCacheManager cacheManager = new SimpleCacheManager();
+    
+    cacheManager.setCaches(internalCaches());
+    return cacheManager;
+  }
+
+  private Collection<Memcached> internalCaches() {
```

Comment:
What is the use of internalCaches? 
Do we have to define all the cache keys here?

Reply:
We need to define which all cache keys are gonna be used with there expiry here